package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-utils-chartutils",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
Include `"sideEffects": false` flag into package.json

Reply:
What is `sideEffects` and why is it required?